packages/wdio-runner/src/index.ts

```diff
@@ -448,7 +448,12 @@ export default class Runner extends EventEmitter {
             })
         }
 
-        await this._browser?.deleteSession()
+        try {
+            await this._browser?.deleteSession()
+        } catch (error) {
+            log.debug('Error deleting session', error)
```

Comment:
Can we extend this error message, to warn users that they potentially have to shut down the browser manually? Also I was wondering if we might want to restrict this behavior only to certain error messages, wdyt?

Reply:
Don't think I have sufficient knowledge to comment.

Happy to update if you provide requirements but I think the change as-is is an improvement over the status quo - no report - unless there's additional effects that I'm missing.